plugins/net_plugin/net_plugin.cpp

```diff
@@ -1499,7 +1501,7 @@ namespace eosio {
 
    // called from connection strand
    void connection::blk_send_branch( const block_id_type& msg_head_id ) {
-      uint32_t head_num = my_impl->get_chain_head_num();
+      uint32_t head_num = my_impl->get_fork_head_num();
```

Comment:
Why change to `get_fork_head_num`? Several other places also change to fork_head. PR description says using chain_head.

Which change is for switching to chain_head? I could not find it.

Reply:
See `sync_recv_block`.
There is now both `get_chain_head_num` and `get_fork_head_num`.